tests/raw_query/tests.py

```diff
@@ -247,46 +248,46 @@ def test_query_representation(self):
         """
         Test representation of raw query with parameters
         """
-        query = "SELECT * FROM raw_query_author WHERE last_name = %(last)s"
-        qset = Author.objects.raw(query, {"last": "foo"})
+        query = [{"$match": {"last_name": "%(last)s" % {"last": "foo"}}}]
```

Comment:
This test seems useless if there is no need for a `params` argument to `raw_mql()`. Using string interpolation to construct the query is not adding any value, so you might just skip test like these if they aren't applicable.

Reply:
I'm not sure I understand why that would be needed with `MQL` or `SQL`. There is another test that includes `params` kwarg (are we planning to support this?) This one seems to be testing string interpolation for no reason that I understand …

Reply:
Query parameters are passed separately in SQL-land [to protect against SQL injection attacks](https://docs.djangoproject.com/en/5.1/topics/security/#sql-injection-protection).